src/libhirte/common/cfg.c

```diff
@@ -338,3 +339,55 @@ const char *cfg_dump(struct config *config) {
         }
         return cfg_info;
 }
+
+int cfg_agent_def_conf(struct config *config) {
+        char *buff = NULL;
+
+        // NodeName
+        if (cfg_set_value(config, CFG_NODE_NAME, get_hostname()) != 0) {
+                return -1;
+        }
+
+        // ManagerHost
+        if (cfg_set_value(config, CFG_MANAGER_HOST, HIRTE_DEFAULT_HOST) != 0) {
+                return -1;
+        }
+
+        // HeartbeatInterval
+        if (asprintf(&buff, "%li", AGENT_HEARTBEAT_INTERVAL_MSEC) <= 0) {
```

Comment:
I think it would be better to change  `AGENT_HEARTBEAT_INTERVAL_MSEC` definition into string and just set it into `CFG_HEARTBEAT_INTERVAL`, no need to perform a conversion here